collaboration-service/config/cache.js

```diff
@@ -1,8 +1,19 @@
 const { createClient } = require('redis')
 
-const redisClient = createClient({
-  url: process.env.REDIS_URL || 'redis://localhost:6379',
-})
+let redisClient = null
+if (process.env.ENV === 'PROD') {
+  redisClient = createClient({
+    socket: {
+      host: 'redis-10745.c299.asia-northeast1-1.gce.cloud.redislabs.com',
+      port: 10745,
+    },
+    password: 'HwDKiJnZH8oceNGS6K76sakPj08mDldK',
```

Comment:
It's a bad idea to have password here, maybe take in from env?

Reply:
Agree, will make the changes!